src/scripts/main.js

```diff
@@ -1,12 +1,15 @@
 'use strict';
 
 const largeImg = document.getElementById('largeImg');
-const minImg = document.querySelectorAll('a');
-const minImgArr = [...minImg];
-
-minImgArr.forEach((item, i) => {
-  minImg[i].addEventListener('click', (ev) => {
-    largeImg.src = item.getAttribute('href');
-    ev.preventDefault();
-  });
+const gallery = document.querySelector('.gallery');
+
+gallery.addEventListener('click', (ev) => {
+  const item = ev.target.closest('.list-item');
+
+  ev.preventDefault();
```

Comment:
you don't need preventDefault there, cause there is no default behavior to clicking on div element